packs/hubot/actions/workflows/post_result.yaml

```diff
@@ -0,0 +1,18 @@
+---
+  chain:
+    -
+      name: "format_result"
```

Comment:
The main motivation for this particular change was to get rid of http call to hubot. I've decided not to touch formatting part partly because a) converting the results based on runner type is not necessarily a thing we want clients to implement and I thought it was you who proposed we keep this kind of logic on st2 side, b) because I've decided to solve one problem at a time and c) formatting is the thing @emedvedev is working on.

I'm not intending to merge this PR just now, I just wanted to give you an impression of how it may look like. If you want formatting happen on bot side, sure, we'll sync with @emedvedev and see where our two PRs would meet.